seqr/utils/logging_utils.py

```diff
@@ -85,4 +85,14 @@ def log_model_bulk_update(logger, models, user, update_type, update_fields=None)
         db_update['updateFields'] = list(update_fields)
     logger.info(
         '{} {} {}s'.format(update_type, len(entity_ids), db_entity), user, db_update=db_update)
-    return entity_ids
\ No newline at end of file
+    return entity_ids
+
+
+def log_model_no_guid_bulk_update(logger, models, user, update_type):
+    if not models:
+        return []
```

Comment:
this function has no return value, why would it return something here?

Reply:
Good catch. It is modified from `log_model_bulk_update`, and I forgot to change it after removing the `entity_ids`.